enrolments/tests/utils/utils.py

```diff
@@ -0,0 +1,48 @@
+from faker import Faker
+
+from enrolments.models import Session, Class, Enrolment
+
+fake = Faker()
+Faker.seed(0)
+
+
+def create_test_sessions(num_sessions):
+    sessions = []
+    for i in range(num_sessions):
+        sessions.append(
+            Session(
+                season=Session.SEASON_CHOICES[i % len(Session.SEASON_CHOICES)][1],
+                year=2020 - i // len(Session.SEASON_CHOICES),
+            )
+        )
+
+    return Session.objects.bulk_create(sessions)
```

Comment:
Oh I wish I knew about `bulk_create` earlier.

Reply:
it's pretty neat! saves transactions if ur creating lots of objs